RELEASE-NOTES.md

```diff
@@ -11,11 +27,12 @@
 * Removed `HashArray::indicesAreUpToDate`
 * Removed `HashArray::removeDuplicates`
 * Removed `$acceptDuplicates` feature from `HashArray`
-* Added `clear` to `TermList`, `AliasGroupList` and `StatementList`
+* Fixed exceptions in `DispatchingEntityIdParser` and `ItemIdParser` not forwarding the previous
+  exception
```

Comment:
Please separate breaking changes from non-breaking changes as was done in the past